test/conformance/testing/include/uur/fixtures.h

```diff
@@ -372,17 +373,19 @@ struct urQueueTest : urContextTest {
 struct urHostPipeTest : urQueueTest {
     void SetUp() override {
         UUR_RETURN_ON_FATAL_FAILURE(urQueueTest::SetUp());
-        UUR_RETURN_ON_FATAL_FAILURE(
-            uur::KernelsEnvironment::instance->LoadSource("foo", il_binary));
-        ASSERT_SUCCESS(uur::KernelsEnvironment::instance->CreateProgram(
-            platform, context, device, *il_binary, nullptr, &program));
 
         size_t size = 0;
-        ASSERT_SUCCESS(urDeviceGetInfo(
+        UUR_ASSERT_SUCCESS_OR_UNSUPPORTED(urDeviceGetInfo(
```

Comment:
this isn't an optional query, adapters should return `false` if they don't support pipes

Reply:
Right, looks like only L0 didn't implement this. I added the missing query support.